### ISSUE TITLE
10.0.2 Errors

### ISSUE THREAD

**Issue by eltreum0** [mpstark/DynamicCam#65]
In 10.0.2 the hide module is causing this error:

```LUA
1x Interface/AddOns/DynamicCam/UiHideModule.lua:341: GameTooltip doesn't have a "OnTooltipSetItem" script
[string "=[C]"]: in function `HookScript'
[string "@Interface/AddOns/DynamicCam/UiHideModule.lua"]:341: in main chunk

Locals:
(*temporary) = GameTooltip {
 0 = <userdata>
 SetQuestLogRewardSpell = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetQuestPartyProgress = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 ProcessLineData = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:303
 SetPvpTalent = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetRuneforgeResultItem = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetHyperlink = <function> defined =[C]:-1
 SetUnit = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetAzeriteEssence = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetVoidDepositItem = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetVoidWithdrawalItem = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetInfoBackdropStyle = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:338
 shoppingTooltips = <table> {
 }
 SetSocketGem = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 updateTooltipTimer = 0.200000
 ItemTooltip = Frame {
 }
 StatusBar = GameTooltipStatusBar {
 }
 SetLootCurrency = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetSlottedKeystone = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetWorldCursor = <function> defined @Interface/FrameXML/GameTooltip.lua:952
 SetLootItem = <function> defined =[C]:-1
 SetMerchantCostItem = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 GetUnit = <function> defined @Interface/FrameXML/GameTooltip.lua:994
 SetOwnedItemByID = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetItemInteractionItem = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetQuestLogSpecialItem = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 TextRight1 = GameTooltipTextRight1 {
 }
 SetEnhancedConduit = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetRecipeReagentItem = <function> defined =[C]:-1
 SetQuestCurrency = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 supportsDataRefresh = true
 SetWeeklyReward = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetMerchantItem = <function> defined =[C]:-1
 SetPvpBrawl = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetAchievementByID = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 BottomOverlay = Texture {
 }
 SetSpellByID = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 supportsItemComparison = true
 SetSocketedRelic = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 ProcessInfo = <function> defined =[C]:-1
 ProcessLines = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:293
 NineSlice = Frame {
 }
 SetInventoryItem = <function> defined =[C]:-1
 RefreshData = <function> defined @Interface/FrameXML/GameTooltip.lua:933
 SetTransmogrifyItem = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetLootRollItem = <function> defined =[C]:-1
 OnLoad = <function> defined @Interface/FrameXML/GameTooltip.lua:927
 SetQuestLogCurrency = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetUnitDebuffByAuraInstanceID = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetBagItem = <function> defined =[C]:-1
 GetSpell = <function> defined @Interface/FrameXML/GameTooltip.lua:990
 SetAction = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetItemKey = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetCompanionPet = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetInventoryItemByID = <function> defined @Interface/SharedXML/Tooltip/TooltipDa```

**Comment by sthdevilish** [comment]
5x Interface/AddOns/DynamicCam/UiHideModule.lua:341: GameTooltip doesn't have a "OnTooltipSetItem" script
[string "=[C]"]: in function `HookScript'
[string "@Interface/AddOns/DynamicCam/UiHideModule.lua"]:341: in main chunk

Locals:
(*temporary) = GameTooltip {
 0 = <userdata>
 SetQuestLogRewardSpell = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetQuestPartyProgress = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 ProcessLineData = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:303
 SetPvpTalent = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetRuneforgeResultItem = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetHyperlink = <function> defined =[C]:-1
 SetUnit = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetAzeriteEssence = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetVoidDepositItem = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetVoidWithdrawalItem = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetInfoBackdropStyle = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:338
 shoppingTooltips = <table> {
 }
 SetSocketGem = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 updateTooltipTimer = 0.200000
 ItemTooltip = Frame {
 }
 StatusBar = GameTooltipStatusBar {
 }
 SetLootCurrency = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetSlottedKeystone = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetWorldCursor = <function> defined @Interface/FrameXML/GameTooltip.lua:952
 SetLootItem = <function> defined =[C]:-1
 SetMerchantCostItem = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 GetUnit = <function> defined @Interface/FrameXML/GameTooltip.lua:994
 SetOwnedItemByID = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetItemInteractionItem = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetQuestLogSpecialItem = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 TextRight1 = GameTooltipTextRight1 {
 }
 SetEnhancedConduit = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetRecipeReagentItem = <function> defined =[C]:-1
 SetQuestCurrency = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 supportsDataRefresh = true
 SetWeeklyReward = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetMerchantItem = <function> defined =[C]:-1
 SetPvpBrawl = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetAchievementByID = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 BottomOverlay = Texture {
 }
 SetSpellByID = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 supportsItemComparison = true
 SetSocketedRelic = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 ProcessInfo = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:236
 ProcessLines = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:293
 NineSlice = Frame {
 }
 SetInventoryItem = <function> defined =[C]:-1
 RefreshData = <function> defined @Interface/FrameXML/GameTooltip.lua:933
 SetTransmogrifyItem = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetLootRollItem = <function> defined =[C]:-1
 OnLoad = <function> defined @Interface/FrameXML/GameTooltip.lua:927
 SetQuestLogCurrency = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetUnitDebuffByAuraInstanceID = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetBagItem = <function> defined =[C]:-1
 GetSpell = <function> defined @Interface/FrameXML/GameTooltip.lua:990
 SetAction = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetItemKey = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetCompanionPet = <function> defined @Interface/SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetInventoryItemByID = <function

5x Interface/AddOns/DynamicCam/UiHideModule.lua:1 (null)

5x FrameXML\Bindings.xml:1 Interface/AddOns/DynamicCam/UiHideModule.lua:1 (null)

**Comment by LudiusMaximus** [comment]
Thank you! Should be fixed with the most recent version 2.0.1.
